client/my-sites/plugins/style.scss

```diff
@@ -324,35 +326,62 @@ body.is-section-plugins header .select-dropdown__item {
 	color: var( --studio-gray-60 );
 	margin-bottom: 8px;
 }
+.plugins__page-title-container {
+	display: none;
+	@include breakpoint-deprecated( '>660px' ) {
+		display: block;
+	}
+}
 .plugins__jetpack-cloud {
 	.plugins__main-header .section-nav {
-		border-width: 0;
+		border-width: 1px;
+		@include breakpoint-deprecated( '>660px' ) {
+			border-width: 0;
+		}
 	}
 	.plugins__main-header {
-		margin: 0;
+		margin: 9px 0;
+		@include breakpoint-deprecated( '>660px' ) {
+			margin: 0;
+		}
 	}
 }
 .plugins__top-container {
 	margin: 0 -32px;
-	padding: 0 48px;
-	border-bottom: 1px solid var( --color-primary-5 );
+	padding: 8px 48px 0;
+	@include breakpoint-deprecated( '>660px' ) {
+		padding: 16px 48px 0;
+		border-bottom: 1px solid var( --color-primary-5 );
+	}
+	@include break-large() {
+		padding: 6px 48px 0;
+	}
 }
 .plugins__content-wrapper {
 	max-width: 1500px;
 	margin: auto;
-	padding: 0;
+	@include break-large() {
+		padding: 0;
+	}
 }
 .plugins__main-content {
 	// We need these negative margin values because we want to make the container full-width,
 	// but our element is inside a limited-width parent.
-	margin: 0 -32px -32px;
-	padding: 16px 48px;
+	margin: 0 -32px;
+	padding: 0 48px;
 	min-height: 100vh;
```

Comment:
I know you didn't add this line in this PR, but I wonder if you know the purpose of it. On the desktop, it extends the vertical height beyond what's necessary.

Reply:
I see. The 100vh is applied to this because we have a different background color to the content, which is not as same as the body color. Removing that cause the below issue

<img width="1651" alt="Screenshot 2022-08-12 at 9 50 42 AM" src="https://user-images.githubusercontent.com/10586875/184284822-ae779573-833d-45c2-912e-07d308efeaee.png">

I'll see how we can fix this so that the unnecessary scroll is not shown.

Reply:
Understood. Thanks for the explanation.